Common/Data/UniverseSelection/OptionChainUniverse.cs

```diff
@@ -42,25 +41,48 @@ public class OptionChainUniverse : Universe
         private readonly TimeSpan _minimumTimeInUniverse = TimeSpan.FromMinutes(15);
         private readonly Dictionary<Symbol, DateTime> _addTimesBySymbol = new Dictionary<Symbol, DateTime>();
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OptionChainUniverse"/> class
+        /// </summary>
+        /// <param name="option">The canonical option chain security</param>
+        /// <param name="universeSettings">The universe settings to be used for new subscriptions</param>
+        /// <param name="liveMode">True if we're running in live mode, false for backtest mode</param>
+        public OptionChainUniverse(Option option,
+            UniverseSettings universeSettings,
+            bool liveMode)
+            : base(option.SubscriptionDataConfig)
+        {
+            Option = option;
+            _underlyingSymbol = new[] { Option.Symbol.Underlying };
+            _universeSettings = universeSettings;
+            _liveMode = liveMode;
+        }
+
         /// <summary>
         /// Initializes a new instance of the <see cref="OptionChainUniverse"/> class
         /// </summary>
         /// <param name="option">The canonical option chain security</param>
         /// <param name="universeSettings">The universe settings to be used for new subscriptions</param>
         /// <param name="securityInitializer">The security initializer to use on newly created securities</param>
         /// <param name="liveMode">True if we're running in live mode, false for backtest mode</param>
+        [Obsolete("This constructor is obsolete because SecurityInitializer is obsolete and will not be used.")]
         public OptionChainUniverse(Option option,
                                    UniverseSettings universeSettings,
                                    ISecurityInitializer securityInitializer,
                                    bool liveMode)
             : base(option.SubscriptionDataConfig, securityInitializer)
         {
-            _option = option;
-            _underlyingSymbol = new[] { _option.Symbol.Underlying };
+            Option = option;
+            _underlyingSymbol = new[] { Option.Symbol.Underlying };
             _universeSettings = universeSettings;
             _liveMode = liveMode;
         }
 
+        /// <summary>
+        /// The canonical option chain security
+        /// </summary>
+        public readonly Option Option;
```

Comment:
for public members, please use properties, in this case, it can be get-only:
``` csharp
public Option Option { get; }
```

This same comment applies to the `Future` class